android/library/src/main/res/drawable-v21/default_button_repeatable_drawable.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:state_pressed="false">
+    <item android:state_enabled="false">
```

Comment:
I would divide drawables by file like we did in `lexicon_button_selector.xml` for example:

```
<?xml version="1.0" encoding="utf-8"?>
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:drawable="@drawable/lexicon_button_pressed_drawable" android:state_enabled="false"/>
    <item android:drawable="@drawable/lexicon_button_pressed_drawable" android:state_focused="true" android:state_pressed="true"/>
    <item android:drawable="@drawable/lexicon_button_pressed_drawable" android:state_focused="false" android:state_pressed="true"/>
    <item android:drawable="@drawable/lexicon_button_drawable" android:state_focused="true"/>
    <item android:drawable="@drawable/lexicon_button_drawable" android:state_focused="false" android:state_pressed="false"/>
</selector>
```